src/components/CopyPopup/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC, useState } from "react";
+import { Popup, PopupWrapper } from "./style";
+
+interface Props extends React.PropsWithChildren<{}> {
+  onClick: () => void;
+}
+
+const CopyPopup: FC<Props> = ({ children, onClick }: Props) => {
+  const [showPopup, setShowPopup] = useState(false);
+
+  const handlePopUp = () => {
+    if (!showPopup) {
+      setShowPopup(true);
+      setTimeout(() => {
+        setShowPopup(false);
+      }, 1000);
```

Comment:
Save this magic number inside a variable named `const DELAY_IN_MILLISECONDS = 1000;`